src/app/components/confirm/index.html

```diff
@@ -5,95 +5,76 @@
 >
 
   <sky-demo-page-summary>
-    The confirm service launches simple confirmation dialogs to allow users to confirm actions. The <stache-code>SkyConfirmService</stache-code> provider lauches the confirmation dialog. Within the service, you may specify the dialog's message and type, and customize the button text.
+    The confirm service launches simple confirmation dialogs to allow users to confirm actions. The <stache-code>SkyConfirmService</stache-code> provider launches the confirmation dialog. Within the service, you can specify the dialog's message and customize the button text.
   </sky-demo-page-summary>
 
-  <sky-demo-page-properties sectionHeading="Service methods">
+  <sky-demo-page-properties sectionHeading="Confirm service methods">
     <sky-demo-page-property
       propertyName="open(config)"
     >
-    Opens a confirmation dialog. The <stache-code>config</stache-code> parameter is of type <stache-code>SkyConfirmConfig</stache-code> and passed to the component's constructor. This method returns the confirmation dialog instance.
+    Opens a confirmation dialog and passes the <stache-code>config</stache-code> parameter is of type <stache-code>SkyConfirmConfig</stache-code> to the component's constructor. This method returns the confirmation dialog instance.
```

Comment:
"...and passes the config parameter is of type..." Sounds strange?

Reply:
Also, would it help to include the specific returned type? "This method returns an object of type `SkyConfirmInstance`." Then, we can document the properties and methods of the `SkyConfirmInstance` in the section below, that is currently titled "Confirmation dialog events".

Reply:
For the first point, I think the problem is just the stray "is" that needed to be deleted. So I deleted that. ... For the second point, I replaced the existing second sentence with your suggestion and changed the "Confirmation dialog events" header to "SkyConfirmInstance events"